clients/fides-js/src/components/fides.css

```diff
@@ -67,6 +67,7 @@
   --fides-overlay-container-border-radius: 12px;
   --fides-overlay-component-border-radius: 4px;
   --fides-overlay-banner-offset: 48px;
+  --fides-banner-font-size-title: var(--16px);
```

Comment:
I removed this variable earlier not realizing the nyt extension was using it, so this is restoring it!